ml/Find_Percentile.py

```diff
@@ -0,0 +1,13 @@
+import numpy
+m = int(input("Enter m"))
+n = int(input("Enter n"))
+p = int(input("Enter p"))
+l = []
```

Comment:
Try to do without using python list.
Only use `numpy array`

Reply:
I have updated the code please check